src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Main.cs

```diff
@@ -0,0 +1,219 @@
+        private PluginInitContext? _context;
+
+        /// <summary>
+        /// The path to the icon for each result.
+        /// </summary>
+        private string _defaultIconPath;
+
+        /// <summary>
+        /// Indicate that the plugin is disposed.
+        /// </summary>
+        private bool _disposed;
+
+        /// <summary>
+        /// List that contain all settings.
+        /// </summary>
+        private IEnumerable<WindowsSetting>? _settingsList;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Main"/> class.
+        /// </summary>
+        public Main()
+        {
+            _assemblyName = Assembly.GetExecutingAssembly().GetName().Name ?? Name;
+            _defaultIconPath = _lightSymbol;
+        }
+
+        /// <summary>
+        /// Gets the localized name.
+        /// </summary>
+        public string Name => Resources.PluginTitle;
+
+        /// <summary>
+        /// Gets the localized description.
+        /// </summary>
+        public string Description => Resources.PluginDescription;
+
+        /// <summary>
+        /// Initialize the plugin with the given <see cref="PluginInitContext"/>.
+        /// </summary>
+        /// <param name="context">The <see cref="PluginInitContext"/> for this plugin.</param>
+        public void Init(PluginInitContext context)
+        {
+            _context = context ?? throw new ArgumentNullException(nameof(context));
+            _context.API.ThemeChanged += OnThemeChanged;
+            UpdateIconPath(_context.API.GetCurrentTheme());
+
+            _settingsList = ResourceHelper.ReadAllPossibleSettings();
+            _settingsList = UnsupportedSettingsHelper.FilterByBuild(_settingsList);
+
+            TranslationHelper.TranslateAllSettings(_settingsList);
+        }
+
+        /// <summary>
+        /// Return a filtered list, based on the given query.
+        /// </summary>
+        /// <param name="query">The query to filter the list.</param>
+        /// <returns>A filtered list, can be empty when nothing was found.</returns>
+        public List<Result> Query(Query query)
+        {
+            if (_settingsList is null)
+            {
+                return new List<Result>(0);
+            }
+
+            var filteredList = _settingsList
+                .Where(Predicate)
+                .OrderBy(found => found.Name);
+
+            var newList = ResultHelper.GetResultList(filteredList, query.Search, _defaultIconPath);
+            return newList;
+
+            bool Predicate(WindowsSetting found)
+            {
+                if (found.Name.Contains(query.Search, StringComparison.CurrentCultureIgnoreCase))
+                {
+                    return true;
+                }
+
+                // Search for Area only by key char
+                if (found.Area.Contains(query.Search.Replace("$", string.Empty), StringComparison.CurrentCultureIgnoreCase)
+                && query.Search.EndsWith("$"))
```

Comment:
Let's switch `$` to `:`. I think this is more intuitive. - Changed my comment after you read it. 😅

Maybe a stupid question: Do we need to respect other cultural differences refard the action char?

Reply:
I don't know, but the wikipedia says
```
Because of its use in early American computer applications such as business accounting,
the dollar sign is almost universally present in computer character sets, ...
```

Reply:
And `:` if we use it? Your thoughts?

Reply:
And I think, this should not be a problem for us
```
The dollar sign was used as a letter in the Turkmen alphabet from 1993 to 1999.
```

Reply:
Sory. I meant the filter char and not the action char.